src/main/java/teammates/logic/api/TaskQueuer.java

```diff
@@ -294,6 +294,21 @@ public void scheduleEmailsForSending(List<EmailWrapper> emails) {
         }
     }
     
+    /**
+     * Schedules production of searchable documents
```

Comment:
So this method and the queue are specifically for creating searchable documents for demo courses and cannot be used for any other thing?
In that case the method/class/queue name should be changed to reflect that as well?. 
On a related note, as long as we are creating queues just for demo course creation, why not put the entire demo course creation in the queue?